### PR TITLE
Complete initial implementation

### PR DESCRIPTION
**Pull request by alexey-rise**
@tejohnso please review.

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tejohnso) [inline .project]: Is this file specific to your text editor? I think we can leave it out.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "@wdio/dot-reporter": "^5.7.8",
+    "@wdio/local-runner": "^5.9.1",
+    "@wdio/mocha-framework": "^5.9.1",
+    "@wdio/sync": "^5.9.1",
+    "eslint": "^5.16.0",
+    "terser": "^4.0.0"
+  },
+  "dependencies": {
+    "chromedriver": "^74.0.0",
+    "http-server": "^0.11.1"
```
Comment (tejohnso) [inline package.json]: I think these can be removed. They're not dependencies of the component. At least, they could be moved into devDependencies, but removing is probably better.

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+          at: .
+      - gcloud/install
+      - gcloud/initialize
+      - run:
+          name: Copy to gcs
+          command: |
+            if [ $CIRCLE_BRANCH=master ];then CHANNEL=stable; else CHANNEL=staging; fi
+            GCS_PATH=gs://widgets.risevision.com/$CHANNEL/components/pricing/pricing-grid-component.mjs
+            gsutil cp build/pricing-grid-component.mjs $GCS_PATH
+            gsutil -m setmeta -r -h "Cache-Control:private, max-age=0" -h "content-type: text/javascript" $GCS_PATH
```
Comment (tejohnso) [inline .circleci/config.yml]: I'm not sure we need to be so cache averse. The others are using public, max-age=300 (recent change).

**Comment by alexey-rise** [comment]
@tejohnso please see the last commit.

**Pull request merged**